actors/builtin/miner/miner_test.go

```diff
@@ -566,6 +566,49 @@ func TestExtendSectorExpiration(t *testing.T) {
 	})
 }
 
+func TestAddLockedFund(t *testing.T) {
+	owner := tutil.NewIDAddr(t, 100)
+	worker := tutil.NewIDAddr(t, 101)
+	workerKey := tutil.NewBLSAddr(t, 0)
+	actor := newHarness(t, tutil.NewIDAddr(t, 1000), owner, worker, workerKey)
+	periodOffset := abi.ChainEpoch(99)
```

Comment:
A better test would be to make this greater than the Quantization to ensure the periodOffset won't push the vesting schedule off by more than the quantization interval (it appears to do this but it's not exercised in the test). Better still would be to test it both ways.

Reply:
ok, to test this it sounds like you also want something like `assert.LessThan(t, newVestingFunds[0], AddFundsEpoch + PledgeVestingSpec.InitialDelay + PledgeVestingSpec.Quantization)` (first vesting epoch < one quantization interval away from initial delay)